web/src/webflow/dropdown.html

```diff
@@ -0,0 +1,864 @@
+        onclick="openDropdown()"
+        oninput="handleOnInput()"
+      />
+      <div id="skf-combo-box-clear-button" onclick="clearIndustrySelection()">
+        <svg width="25" height="25" viewBox="0 0 25 25" xmlns="http://www.w3.org/2000/svg">
+          <path xmlns="http://www.w3.org/2000/svg" d="M0 0h24v24H0z" fill="none" />
+          <path
+            xmlns="http://www.w3.org/2000/svg"
+            d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
+          />
+        </svg>
+      </div>
+      <div id="skf-dropdown-button" onclick="toggleDropdown()" tabindex="0">
+        <svg
+          id="skf-dropdown-arrow"
+          fill="#757575"
+          height="10px"
+          width="10px"
+          version="1.1"
+          id="Layer_1"
+          xmlns="http://www.w3.org/2000/svg"
+          xmlns:xlink="http://www.w3.org/1999/xlink"
+          viewBox="0 0 386.257 386.257"
+          xml:space="preserve"
+        >
+          <polygon points="0,96.879 193.129,289.379 386.257,96.879 " />
+        </svg>
+      </div>
+      <ul id="skf-dropdown" hidden></ul>
+    </div>
+    <a id="skf-combo-box-button">Access My Starter Kit</a>
+  </div>
+</div>
+
+<script>
+  let industries = [];
+  let selectedIndustry = {};
+
+  document.addEventListener("DOMContentLoaded", () => {
+    setIndustryData();
+    populateDropdown(industries);
+  });
+
+  const setIndustryData = () => {
+    const industryNames = Array.from(document.getElementsByClassName("sk-industry-name"));
+    const industryDescriptions = Array.from(document.getElementsByClassName("sk-industry-description"));
+    const industrySynonyms = Array.from(document.getElementsByClassName("sk-industry-synonyms"));
+    const industrySlugs = Array.from(document.getElementsByClassName("sk-industry-slug"));
+
+    industryNames.forEach((name, index) => {
+      const synonyms = industrySynonyms[index].innerText.split(", ").map((synoynm) => synoynm.toLowerCase());
+
+      industries.push({
+        name: name.innerText,
+        description: sanitizeString(industryDescriptions[index].innerText),
+        slug: industrySlugs[index].innerText,
+        synonyms,
+      });
+
+      industries.sort((a, b) => a.name.toLowerCase().localeCompare(b.name.toLowerCase()));
+    });
+
+    addGenericIndustryToList(industries);
+    document.getElementsByClassName("sk-industry-items")[0].toggleAttribute("hidden");
+  };
+
+  const addGenericIndustryToList = (industryList) => {
+    industryList.unshift({
+      name: "All Other Businesses",
+      description: "Select this if you do not see your specific industry",
+      slug: "all-other-businesses",
+      synonyms: [],
+    });
+  };
+
+  const sanitizeString = (string) => {
+    return string.replace("'", "’");
+  };
+
+  const populateDropdown = (industries) => {
+    const dropdown = document.getElementById("skf-dropdown");
+    const input = document.getElementById("skf-input");
```

Comment:
[dust] I feel like my instinct here is to always use ID's to grab elements and always use classes for CSS to select and style things. I feel this division is more robust and more easily changeable.